deepchem/models/torch_models/tests/test_acnn.py

```diff
@@ -98,10 +98,13 @@ def test_atomic_convolution_model():
     features.append(
         (frag1_coords, frag1_nbr_list, frag1_z, frag2_coords, frag2_nbr_list,
          frag2_z, system_coords, system_nbr_list, system_z))
+    features = np.asarray(features, dtype=object)
     labels = np.random.rand(batch_size)
     train = NumpyDataset(features, labels)
     atomic_convnet.fit(train, nb_epoch=150)
-    assert np.allclose(labels, atomic_convnet.predict(train), atol=0.01)
```

Comment:
You are losing a test here; the new code no longer checks that the moddel makes correct predictions. Please restore that test

Reply:
Addressed